nixos/modules/system/boot/timesyncd.nix

```
@@ -2,33 +2,50 @@
 
 with lib;
 
+let
+  cfg = config.services.timesyncd;
+in
 {
 
   options = {
 
-    services.timesyncd = {
+    services.timesyncd = with types; {
```

```suggestion
    services.timesyncd = {
```
This is as worse as with lib over half the file. Keeping the types prefix has no real disadvantage.

---

I personally agree it's more readable like this and should be left as-is.

@SuperSandro2000 this is an instance of https://github.com/NixOS/nixpkgs/pull/336200.

Please refrain from nitpicking solely on formatting matters, it's a energy drain you have repeatedly been made aware of. Please stop.